Sources/SectionScrubber.swift

```diff
@@ -101,14 +117,14 @@ public class SectionScrubber: UIView {
             if self.sectionLabelState != oldValue {
                 if self.sectionLabelState == .Visible { self.setSectionLabelActive() }
                 if self.sectionLabelState == .Hidden { self.setSectionLabelInactive() }
-                self.updateSectionLabelFrame()
             }
         }
     }
 
     private var scrubberState = VisibilityState.Hidden {
```

Comment:
I'm thinking if this wouldn't work better as an override on the `hidden` property, that makes it animatable? 💭 cc @marijnschilling @3lvis